hedera/settings.py

```diff
@@ -330,6 +330,7 @@
 ACCOUNT_EMAIL_CONFIRMATION_REQUIRED = False
 ACCOUNT_EMAIL_CONFIRMATION_EXPIRE_DAYS = 7
 ACCOUNT_USE_AUTH_AUTHENTICATE = True
+ACCOUNT_LOGIN_REDIRECT_URL = "/dashboard"
```

Comment:
This setting handles the redirect on login, but what about signup? Currently, after you sign-up, it redirects you back to the home page, but I'm assuming we'll want to redirect to the dashboard in that case as well. 

It looks like there's a separate [ACCOUNT_SIGNUP_REDIRECT_URL](https://github.com/pinax/django-user-accounts/blob/68bac737dbcd1f5f407aae80d4783b496fcfded8/docs/settings.rst#account_signup_redirect_url) setting that we would need to add.

Reply:
Oh I didnt think about that! Thanks Artie will add that to the code.